tests/Common/DataFixtures/ProxyReferenceRepositoryTest.php

```diff
@@ -96,7 +98,7 @@ public function testReferenceReconstruction(): void
 
         $roleFixture->load($em);
         // first test against managed state
-        $ref = $referenceRepository->getReference('admin-role');
+        $ref = $referenceRepository->getReference('admin-role', Role::class);
```

Comment:
Should we have made these adjustments to the tests on 1.x already?

Reply:
Yes and no, depends on the test strategy.
The "old way" to call getReference must be tested on 1.x too.

On 1.x test should have been duplicated with
- The old way, working but triggering deprecation, and a `@group legacy`
- The new way working without deprecation.

Seems like no test were added/duplicated in https://github.com/doctrine/data-fixtures/pull/409, so only the old way were tested. Now on 2.x tests are updated.

Reply:
So, we're missing coverage for the non-deprecated way in 1.x currently? We should fix that first, I suppose.

Reply:
I did https://github.com/doctrine/data-fixtures/pull/465